tests/api/test_events.py

```diff
@@ -152,6 +153,40 @@ def test_api_update_title(db, api_client, calendar, default_account):
     assert e_put_data['when']['time'] == e_data['when']['time']
 
 
+import pytest
+@pytest.mark.only
+def test_api_pessimistic_update(db, api_client, calendar, default_account):
+    e_data = {
+        'title': '',
+        'calendar_id': calendar.public_id,
+        'when': {'time': 1407542195},
+    }
+
+    e_resp = api_client.post_data('/events', e_data,
+                                  headers={ "X-API-Version": API_VERSIONS[1] })
```

Comment:
No. Stripe uses `Stripe-Version`. Clearbit uses `Api-Version`. Apparently, it's not recommended anymore to append `X-` to HTTP headers, so I'll just go with `Api-Version`.